test/unit/dao/FashionKnex-test.js

```diff
@@ -29,41 +29,59 @@ test.afterEach(async (t) => {
 test('should throw error if finding by id results in wrapping an invalid object', async (t) => {
   const { daoHelper, sandbox } = t.context
 
-  sandbox.stub(daoHelper._knex, 'limit').callsFake(async () => {
-    return Promise.resolve([{
-      type: true
-    }])
-  })
+  sandbox.stub(daoHelper._knex, 'limit').resolves([{
+    type: true
+  }])
 
   let error = await t.throws(daoHelper.findById('abc123'))
   t.is(error.message, 'Error(s) while wrapping model with data "{"type":true}": "type: Invalid value: true"')
-  daoHelper._knex.limit.restore()
 })
 
 test('should throw error if find query throws error', async (t) => {
   const { daoHelper, sandbox } = t.context
 
-  sandbox.stub(daoHelper._knex, 'limit').callsFake(async () => {
-    return Promise.reject(new Error('Test error'))
-  })
+  sandbox.stub(daoHelper._knex, 'limit').rejects(new Error('Test error'))
```

Comment:
Just some cleanup - may as well use the .rejects/.resolves helpers instead of callsFake for simple rejects/resolves.